javascript/eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     "no-shadow-restricted-names": 2,
     "no-undef": 2,
     "no-use-before-define": 2,
+    "quotes": [2, "double", "avoid-escape"],
```

Comment:
It means it won't diagnose a single quote as an error, if the only alternative would have been an escaped double quote.
http://eslint.org/docs/rules/quotes